src/dsp_tools/excel2xml.py

```diff
@@ -1697,10 +1710,8 @@ def write_xml(root: etree.Element, filepath: str) -> None:
         None
     """
     etree.indent(root, space="    ")
-    xml_string = etree.tostring(root, encoding="unicode", pretty_print=True)
-    xml_string = '<?xml version="1.0" encoding="UTF-8"?>\n' + xml_string
-    xml_string = xml_string.replace("&lt;", "<")
-    xml_string = xml_string.replace("&gt;", ">")
+    xml_string = etree.tostring(root, encoding="unicode", pretty_print=True, doctype='<?xml version="1.0" encoding="UTF-8"?>')
+    xml_string = xml_string.replace("\'", "'")
```

Comment:
One new problem popped up: 
```
>>> etree.tostring(excel2xml.make_text_prop(":test", "don't"), encoding="unicode")
'<text-prop name=":test"><text>don\'t</text></text-prop>'
```
If I upload such an XML to DSP, DSP-APP displays the backslash as is. 
If I want to get rid of the backslash, I only see the solution to replace it before writing the file. Do you see a better option?